static/js/publisher/pages/Releases/components/releasesTable/row.tsx

```diff
@@ -6,19 +6,24 @@ import { getRevisionsArchitectures } from "../../helpers";
 import ReleasesTableChannelHeading from "./channelHeading";
 
 // generic releases table row component
-const ReleasesTableRow = (props: {
-  canDrag?: any;
+const ReleasesTableRow = ({
+  canDrag = true,
+  risk,
+  branch,
+  revisions,
+  canDrop,
+  children,
+}: {
+  canDrag: boolean;
   risk?: any;
   branch?: any;
   revisions?: any;
   canDrop?: any;
   children?: any;
 }) => {
-  const { risk, branch, revisions, canDrop, children } = props;
+  const _canDrag = !!revisions && canDrag;
```

Comment:
Can `canDrag` be just overridden if needed? Is the separate `_canDrag` needed?

```suggestion
  canDrag = !!revisions && canDrag;
```